### Request
bool linear_dock         
bool orientation_dock     
float64 distance        
float64 orientation       
---
### Response
bool success   # indicates successful run of triggered service
string message # informational, e.g. for error messages 
